tests/testthat/_snaps/write_dwc/dwc_occurrence.csv

```diff
@@ -1,5 +1,5 @@
 type,license,rightsHolder,datasetID,institutionCode,collectionCode,datasetName,basisOfRecord,dataGeneralizations,occurrenceID,sex,lifeStage,reproductiveCondition,occurrenceStatus,organismID,organismName,eventID,parentEventID,eventType,eventDate,samplingProtocol,eventRemarks,minimumElevationInMeters,maximumElevationInMeters,locationRemarks,decimalLatitude,decimalLongitude,geodeticDatum,coordinateUncertaintyInMeters,scientificNameID,scientificName,kingdom
-Event,https://creativecommons.org/publicdomain/zero/1.0/legalcode,,https://doi.org/10.5281/zenodo.10053903,MPIAB,Movebank,"O_ASSEN - Eurasian oystercatchers (Haematopus ostralegus, Haematopodidae) breeding in Assen (the Netherlands) [subsampled representation]",HumanObservation,,5515851_5642_start,unknown,adult,,present,5515851,,5515851_5642_start,5515851_5642,tag attachment,2018-05-09T14:15:00Z,tag attachment,UvA-BiTS tag attached by harness to free-ranging animal,,,,53.00776667,6.58055556,EPSG:4326,187,urn:lsid:marinespecies.org:taxname:147436,Haematopus ostralegus,Animalia
+Event,https://creativecommons.org/publicdomain/zero/1.0/legalcode,,https://doi.org/10.5281/zenodo.10053903,MPIAB,Movebank,"O_ASSEN - Eurasian oystercatchers (Haematopus ostralegus, Haematopodidae) breeding in Assen (the Netherlands) [subsampled representation]",HumanObservation,,5515851_5642_start,unknown,adult,,present,5515851,,5515851_5642_start,5515851_5642,tag attachment,2018-05-09T14:15:00Z,tag attachment,"UvA-BiTS UvA-BiTS UvA-BiTS UvA-BiTS UvA-BiTS UvA-BiTS UvA-BiTS tag attached by harness harness harness harness harness harness harness to free-ranging animal | Tracker malfunction, tagged with new tracker 5750. Found dead, possibly predated by peregrine falcon. Bird originally tracked with tracker 5635.",,,,53.00776667,6.58055556,EPSG:4326,187,urn:lsid:marinespecies.org:taxname:147436,Haematopus ostralegus,Animalia
```

Comment:
Something seems to go wrong here, a vector of values is attached to a single row.

Reply:
Yes it is my `paste2()` helper function that doesn't work as I hoped it would do. The problem is that `paste()` prints NA, e.g.

> UvA-BiTS NA tag attached by harness to free-ranging animal | NA

Working on a solution :-)